### ISSUE TITLE
tfa.losses.SigmoidFocalCrossEntropy() vs focal_loss() in this repo

### ISSUE THREAD

**Issue by glenn-jocher** [google/automl#163]
@mingxingtan, just a quick question. Is there a reason for implementing the custom focal loss function in this repo:
https://github.com/google/automl/blob/04935a7c1003e80b47aa2b9cbbbe681b6a176120/efficientdet/det_model_fn.py#L123

instead of using the existing tf addons function? I'm new tf, coming from pytorch, so a bit confused. Thank you!
https://www.tensorflow.org/addons/api_docs/python/tfa/losses/SigmoidFocalCrossEntropy

**Comment by mingxingtan** [comment]
Good point @glenn-jocher. I wasn't aware of this utility.  Either one is good to me if they are equivalent, but keeping the explicit version might help people to understand more clearly?

**Comment by glenn-jocher** [comment]
@mingxingtan ah ok, got it! I was just wondering if there was some special magic in this version. I tested both versions and they are mathematically equivalent. Thanks for the explanation!
